package.json

```diff
@@ -57,7 +57,8 @@
     "nx": "^15.0.13",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "rimraf": "^3.0.2"
+    "rimraf": "^3.0.2",
+    "typescript": "5.1.6"
```

Comment:
Why is typescript pinned to 5.1.6 in this PR? You have `^5.4.2` in the open source repo ([example](https://github.com/godaddy/gasket/blob/main/packages/gasket-plugin-analyze/package.json#L55))

Reply:
I'm happy to continue to use latest version of typescript, but didn't want to go breaking the typescript-eslint.

Reply:
If it is just a warning and still works, given we can hopefully just remove `gasket-typescript-tests` once these typechecks are added directly to each package, I'd prefer to see us keep up with latest if possible.

Reply:
10-4